app/views/themes/dc_repository/_footer.html.erb

```diff
@@ -0,0 +1,17 @@
+<footer class="navbar navbar-inverse site-footer">
```

Comment:
Should the header also be in this directory?

Reply:
The top header is comprised of _masthead.html.erb, _logo.html.erb and _user_util_links.html.erb. Here is the [PR](https://github.com/scientist-softserv/utk-hyku/commit/47a44044a263123bd8bbab4398c0c3454e197b1d#diff-b2d655c9df0b3f1ba15f24c5a34aea9ee6a293be16b71361c3f0b169ac4756f2) for the changes made for creating the header.

Reply:
ah gotcha. thanks for clarifying.